src/routes/safe/components/AddressBook/ImportEntryModal/index.tsx

```diff
@@ -92,16 +91,19 @@ const ImportEntryModal = ({ importEntryModalHandler, isOpen, onClose }) => {
   const validateCsvData = (data) => {
     for (let index = 0; index < data.length; index++) {
       const entry = data[index]
-      if (!entry.data[0] || !entry.data[1]) {
+      if (!entry.data[0] || !entry.data[1] || !entry.data[2]) {
```

Comment:
Are they always strings? What if chainId is 0?

Reply:
react-papaparse parses the cells as strings